### DISCUSSION TITLE
细节有些不足

### DISCUSSION THREAD

**Discussion by iloveyou01-C** [amll-dev/applemusic-like-lyrics#277] [Ideas]
我发现歌词在滚动的时候是全部被向上拉起来的，而apple music不是，他是以显示的歌词为中心，在这句歌词结束后向上推上面的歌词，把下面的歌词往上拉。不知道能不能实现这个点

**Comment by HKLHaoBin** [comment]
不好看，还是整体向上拉吧

**Comment by Steve-xmh** [comment]
官方AM动画对每个歌词行间的延迟调的比较大，AMLL弱化了这个延迟以显连贯，但**不是完全没有**这个依次被连带拉起的效果
